tools/flang1/flang1exe/semant3.c

```diff
@@ -541,6 +541,10 @@ semant3(int rednum, SST *top)
       if (CLASSG(sptr) && !MONOMORPHICG(sptr) && !ALLOCATTRG(sptr)) {
         error(1217, ERR_Severe, gbl.lineno, SYMNAME(sptr), CNULL);
       }
+      if (STYPEG(sptr) == ST_ENTRY && FVALG(sptr)) {
+        error(72, 3, gbl.lineno, SYMNAME(sptr), "- should use result-name");
```

Comment:
can we add actual variable name instead of result-name here?. 

 if not possible,  can we remove  "should use result-name"

error message will be 
PGF90-S-0072-Assignment operation illegal to ent

Reply:
This patch has been updated and we can report errors with the actual variable name now.